tools/travis/build_snapcraft_snap.sh

```diff
@@ -28,9 +28,14 @@ lxc="/snap/bin/lxc"
 # Workaround for
 # - Setup snap "core" (2462) security profiles (cannot reload udev rules: exit status 2
 $lxc exec snap-builder -- sh -c "snap install core" || echo "ignored error"
+# Simplify the version to use it in the following stages.
+sed -i "s/version: git/version: test/g" "$project_path/snap/snapcraft.yaml"
```

Comment:
If you are doing this to have a consistent filename, let me suggest using `--output` instead.

Reply:
good call.